### DISCUSSION TITLE
How one should handle triggers in this case?

### DISCUSSION THREAD

**Discussion by NowinskiK** [Azure-Player/azure.datafactory.tools#281] [Q&A] [answered]
How one should handle triggers in this case? Let say, we move all but the trigger to UAT/PRD but they have their own triggers which should not be disturbed when moving objects from DEV.

**Comment by NowinskiK** [comment, marked as answer]
In such a case, you must use a selective deployment, which means you determine which objects should be published. The DevOps extension and PS module I created a few months ago allow you to do so. You can set up a different set of objects to be deployed per environment. That's not a problem.

**Comment by namanpatel12** [comment]
How can I change schedule like hours or weekdays in config file when deploying to other data factory? It seems like something is not right in syntax or something else.

trigger,xyz,typeProperties.recurrence.schedule.hours,17
trigger,abc,typeProperties.recurrence.schedule.weekDays,"Friday"

tried above but no luck. Getting below error.

![image](https://user-images.githubusercontent.com/84165941/227080841-ab5bf445-1bb3-45da-b6b3-49a0f70efb95.png)


It will great help if you can direct. @NowinskiK

**Reply by NowinskiK** [reply]
Check this thread: https://github.com/Azure-Player/azure.datafactory.tools/discussions/193

**Reply by namanpatel12** [reply]
Thanks Kamil
